src/OneExplorer.ts

```diff
@@ -0,0 +1,183 @@
+    this._onDidChangeTreeData.fire();
+  }
+
+  getTreeItem(element: OneNode): vscode.TreeItem {
+    element.command = { command: 'oneExplorer.openFile', title: "Open File", arguments: [element.node] };
+    return element;
+  }
+
+  getChildren(element?: OneNode): OneNode[]|Thenable<OneNode[]> {
+    if (this.cfgMap === undefined) {
+      vscode.window.showInformationMessage('No ONE model or config in empty workspace');
+      return Promise.resolve([]);
+    }
+
+    if (element) {
+      return Promise.resolve(this.getNode(element.node));
+    } else {
+      return Promise.resolve(this.getNode(this.cfgMap));
+    }
+  }
+
+  private getNode(node: Node): OneNode[] {
+    const toOneNode = (node: Node): OneNode => {
+      if (node.childs.length > 0) {
+        return new OneNode(node.name, vscode.TreeItemCollapsibleState.Collapsed, node);
+      } else {
+        return new OneNode(node.name, vscode.TreeItemCollapsibleState.None, node);
+      }
+    };
+
+    return node.childs.map(node => toOneNode(node));
+  }
+
+  private getConfigMap(rootPath: vscode.Uri): Node {
+    const node = {type: NodeType.directory, name: path.parse(rootPath.fsPath).base, dir: true, childs: [], uri: rootPath};
+    this.searchConfigs(node, path.dirname(rootPath.fsPath));
+    return node;
+  }
+
+  private searchConfigs(node: Node, dirPath: string) {
+    const dirpath = path.join(dirPath, node.name);
+    const files = fs.readdirSync(dirpath);
+
+    for (const fn of files) {
+      const fpath = path.join(dirpath, fn);
+      const fstat = fs.statSync(fpath);
+
+      if (fstat.isDirectory()) {
+        const dirNode = {type: NodeType.directory, name: fn, dir: true, childs: [], uri: vscode.Uri.file(fpath)};
+
+        this.searchConfigs(dirNode, dirpath);
+        if (dirNode.childs.length > 0) {
+          node.childs.push(dirNode);
+        }
+      }
+      else if (fstat.isFile() 
+        && (fn.endsWith('.pb') || fn.endsWith('.tflite') || fn.endsWith('.onnx'))) {
+        const modelNode: Node = {type: NodeType.model, name: fn, childs: [], uri: vscode.Uri.file(fpath)};
+
+        this.searchPairConfig(modelNode, dirpath);
+
+        node.childs.push(modelNode);
+      }
+    }
+  }
+
+  /**
+   * Search cfg files in the same directory of the node
+   * 
+   * NOTE It assumes 1-1 relation for model and config
+   * 
+   * TODO(dayo) Support N-N relation
+   * TODO(dayo) Search by parsing config file's model entry
+   */
+  private searchPairConfig(node: Node, dirPath: string)
+  {
+    const dirpath = path.dirname(path.join(dirPath, node.name));
+    const files = fs.readdirSync(dirpath);
+
+    const extSlicer = (fileName: string) =>
+    {
+      return fileName.slice(0, fileName.lastIndexOf('.'));
+    };
+
+    for (const fn of files) {
+      const fpath = path.join(dirpath, fn);
+      const fstat = fs.statSync(fpath);
+
+      if (fstat.isFile() 
+      && fn.endsWith('.cfg')
+      && (extSlicer(fn) === extSlicer(node.name))){
```

Comment:
I wonder it assume that cfg file name and model file name are same.

Reply:
I have added 'todo' to parse .cfg file's model file name. I think we need cfg file importer to be implemented for the feature.

Reply:
Could you add more into TODO?

`TODO(dayo) Search by parsing config file's model entry (Currently model name and cfg name must match)`

Without knowing `(Currently model name and cfg name must match)`,
`(extSlicer(fn) === extSlicer(node.name))` was hard to understand since I didn't know your intention. 🐱 

By the way, this approach is quick and reasonable for demo. :-)

Reply:
Applied :-D